x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch.scss

```diff
@@ -5,6 +5,14 @@
   }
 }
 
+.lnsChartSwitch__options {
+  .euiSelectableListItem__append {
+    flex-grow: 1;
+    display: flex;
+    align-items: baseline;
+  }
+}
```

Comment:
I think these styles may be able to be removed after applying my aforementioned changes. 

If for some reason we still need to apply styles though, we need to avoid targeting EUI classes if possible.

Reply:
Styles overrides still required with `EuiBadge`. 😞 
I also didn't find a way to pass a custom class to the `append` element. Looks like it's only possible for the entire `li` element.

Reply:
I took a deeper look at this. Looks like the append is expecting an inline element. Rather than using the above styles and targeting the `.euiSelectableListItem__append` class, try applying a style of `display: inline-flex;` to the offending `EuiFlexGroup` (located at [line 300](https://github.com/elastic/kibana/pull/97978/files#diff-e54618456c0c8c18a47f328670aa0fb52fc67fffe1347525973e68e76d6848a7R300)).

Reply:
Thanks for the suggestion @MichaelMarcialis , I added these changes in https://github.com/elastic/kibana/pull/97978/commits/d699e9e5db9e74cf9daedb7dd29f0b6cb961d205